homeassistant/components/ista_ecotrend/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: The integration registers no actions.
+  appropriate-polling: done
+  brands: done
+  common-modules:
+    status: done
+    comment: Has coordinator for centralized data retrieval. No need for base entity as there is only one platform.
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: The integration registers no actions.
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: The integration registers no events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions:
+    status: exempt
+    comment: The integration registers no actions.
+  config-entry-unloading: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: Integration has no configuration parameters
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: done
+  reauthentication-flow: done
+  test-coverage: done
+
+  # Gold
+  devices: done
+  diagnostics: todo
+  discovery-update-info:
+    status: exempt
+    comment: The integration is a web service, there are no discoverable devices.
+  discovery:
+    status: exempt
+    comment: The integration is a web service, there are no discoverable devices.
```

Comment:
How does the device communicate with the webservice?

Reply:
You mean the metering devices? They report with some proprietary/encrypted radio signal to a central unit in the stairwell. And that unit submits the collected data to ista. But there is no link between the webservice and the individual devices. The webservice only gives you a monthly total consumption.

Reply:
But does that go via your local network?

Reply:
nope, completely independent. The central unit in the stairwell collects data from all apartments in the building, it is in no ones network. I assume it reports to ista with a built in cellular modem.